### DISCUSSION TITLE
Is it possible to use '--single-directory' with '--sync-shared-folders' ?

### DISCUSSION THREAD

**Discussion by wn2000** [abraunegg/onedrive#1315] [Q&A] [answered]
Hello is it possible to use `--single-directory` with `--sync-shared-folders` to sync a single directory from OneDrive Business Shared Folders?
I tried that and it says the specified single directory does not exist on OneDrive.

**Comment by abraunegg** [comment, marked as answer]
@wn2000 
> Hello is it possible to use `--single-directory` with `--sync-shared-folders` to sync a single directory from OneDrive Business Shared Folders?
> I tried that and it says the specified single directory does not exist on OneDrive.

It should be possible, however you need to ensure that the client is correctly configured and test with `--dry-run` first.

Also ensure you are running the latest client version as per https://github.com/abraunegg/onedrive/releases as there was an [issue](https://github.com/abraunegg/onedrive/issues/1234) where your exact use case highlighted a bug, which is fixed in the latest release.
